Document limitation on reinit-ing runs with a deleted ID (#1117)
wandb/docs: content/guides/models/track/runs/_index.md

4. Choose the **Delete** button (trash can icon) above the table.
5. From the modal that appears, choose **Delete**.

{{% alert %}}

> Once a run with a specific ID is deleted, it may not be used as the id for another run for at least 7 days. This is due to W&B allowing you to recover accidentaly deleted runs up to 7 days after their deletion.

Reviewer: ```suggestion
Once a run with a specific ID is deleted, it may not be used as a run ID for 7 days. During that time, the deleted run remains recoverable. After 7 days, the run is permanently deleted and the ID may be used again.
```

> Newer versions of the SDK (0.19.7+) display an error message if you attempt to initiate a run that has been deleted recently.

Reviewer: ```suggestion
```

4. Choose the **Delete** button (trash can icon) above the table.
5. From the modal that appears, choose **Delete**.

{{% alert %}}

> Once a run with a specific ID is deleted, it may not be used as a run ID for 7 days. During that time, the deleted run remains recoverable. After 7 days, the run is permanently deleted and the ID may be used again.

Reviewer: ```suggestion
Once a run with a specific ID is deleted, its ID may not be used on another run for 7 days. During that time, the deleted run remains recoverable. After 7 days, the run is permanently deleted and the ID can be used again.
```
